SessionUIKit/Style Guide/Themes/Theme+Colors.swift

```diff
@@ -116,9 +116,9 @@ internal extension UIColor {
 }
 
 public extension UIColor {
-    static let primary: UIColor = UIColor(dynamicProvider: { _ in
-        return ThemeManager.primaryColor.color
-    })
+    /// This value shouldn't be used dirrectly, it should be resolved via the `ThemeManager.color(for:in:with:)` function, due
```

Comment:
@mpretty-cyro This change break all the theme colours that returns `.primary`. I've changed this extension to be `internal` so it won't be used directly outside of theming. 
If we want to refactor this, we'll need to refactor the whole theming colours. It needs to be tracked as a seperate project in my opinion.